raisinbread/test/api/LorisApiProjectsTest.php

```diff
@@ -20,6 +20,8 @@ class LorisApiProjectsTest extends LorisApiAuthenticatedTest
 {
     protected $projectName = "Pumpernickel";
 
+    protected $instrumentName = "testtest";
```

Comment:
@regisoc please using "aosi" in the test case, or keep the markTestSkipped in this  test case. I will figure out the missing bmi in docker later.